module-2/newspaper/styles.css

```diff
@@ -2,74 +2,127 @@
     box-sizing: border-box;
 }
 body {
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: 'Helvetica Neue','Arial Nova',Helvetica,Arial,sans-serif;
+}
+.top-container {
+    position: sticky;
+    top: 0;
+    z-index: 100;
 }
-/*.container{
-    display: flex;
-}*/
 .header {
     display: flex;
-    flex-wrap: nowrap;
-    flex-direction: row;
+    flex-flow: row nowrap;
     justify-content: space-between;
-    background-color: grey;
+    align-items: center;
+    background-color: white;
 }
-.header>div {
-    background-color: grey;
+.header > div {
     text-align: center;
-    font-size: 20px;
+    font-size: 15px;
 }
 .nav-bar {
     display: flex;
-    flex-wrap: nowrap;
-    flex-direction: row;
+    flex-flow: row nowrap;
     justify-content:space-around;
-    background-color:black;
+    background-color:black; 
 }
-.nav-bar>div{
-    justify-content: space-between;
+.nav-bar > div{
+    flex-flow: row nowrap;
+    justify-content: center;
     background-color: black;
     color: antiquewhite;
     text-align: center;
     font-size: 15px;
 }
+a:link{
+    color: white;
+}
+a:visited{
+    color: white;
+}
+a:hover{
+    color:bisque;
+    background-color: black;
+}
+a:active{
+    color: white;
+}
+.both-columns{
+    display: flex;
+    justify-content: center;
+}
 .left-column{
-    flex: 1;
+    display: inline-block;
+    float: left;
+    margin-left: 10px;
+    width: 700px;
 }
 .right-column{
-    flex: 3;
+    display: inline-block;
+    float: right;
+    width: 300px;
+    flex-shrink: 100;
+    margin-left: 30px;
 }
-.flex-container3 {
-    flex-direction: column;
-    width: 70%;
-    flex-wrap: wrap;
+.big-headline {
+    flex-flow: column wrap;
+    padding-right: 5px;
+    border-right: 1px solid grey;
     margin-top: 10px;
+    margin-bottom: 30px;
 }
-.flex-container3>div{
+.big-headline > div{
     font-size: 40px;
+    font-weight: bolder;
```

Comment:
You can also use numerical values (100 through 900, only 100's) to represent font-weight.. I'd recommend those over 'bolder', since you'll have more granular control: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight.